pkg/executor/docker/gateway/gateway.sh

```diff
@@ -29,12 +29,6 @@ while IFS= read -r BRIDGE_SUBNET; do
     iptables -A INPUT -p tcp --src "${BRIDGE_SUBNET}" --dport 8080 -j ACCEPT
 done < <(cat /etc/bacalhau/allowed-clients.txt)
 
-# Apply rate limits to the outbound connections. We just do this for all
-# interfaces rather than working out which is our Internet connection.
-while IFS= read -r IFACE; do
-    tc qdisc add dev "${IFACE}" root tbf rate 10mbit burst 32kbit latency 10sec
-done < <(ip --json address show | jq -rc '.[] | .ifname')
-
```

Comment:
do we need to remove these even if we are skipping the tests? If yes, then please add an issue to make sure we fix this way before 1.5

Reply:
Done. We don't need to remove this. So reverted it.